lib/subjects.js

```diff
@@ -104,7 +104,13 @@ module.exports = function (app, _private = null) {
               }
             } else {
               // Match was on a variant- use that in the response.
-              const lowerCaseMatchedVariantTerm = hit.highlight['variants.keyword'][0]
+              let highlight
+              if ('variants' in hit.highlight) {
+                highlight = 'variants'
+              } else {
+                highlight = 'variants.keyword'
+              }
+              const lowerCaseMatchedVariantTerm = hit.highlight[highlight][0].toLowerCase()
               const properlyFormattedVariantTerm = hit._source.variants.find((formattedVariant) => {
                 if (formattedVariant.toLowerCase() === lowerCaseMatchedVariantTerm) {
```

Comment:
this is totally a nitpick, but it might be a little clearer to future readers have the lower casing of both of these terms happen on the same line. ie rename `lowerCasedMatchedVariantTerm` to `matchedVariantTerm` and move the `toLowerCase` invocation to the if clause.

Reply:
it might also be worth calling out in a comment what the expected casing state is for keyword vs text matches.